advanced-creation.md

```diff
@@ -220,7 +220,7 @@ const noUserAgent = got.extend({
 
 ```js
 const httpbin = got.extend({
-	baseUrl: 'https://httpbin.org/'
+	prefixUrl: 'https://httpbin.org/'
```

Comment:
This is broken for now. See https://github.com/sindresorhus/got/pull/829/files#r302707096